src/scss/components/checkbox/_index.scss

```diff
@@ -80,8 +80,9 @@ hx-checkbox-control {
 
 // ========== PRISTINE ==========
 hx-checkbox {
+  background-color: $gray-0;
   background-color: var(--hxCheckbox-bgcolor, $gray-0);
-  border-radius: 2px;
+  border: $gray-500;
```

Comment:
this line is a border short hand with just the color var. The line 86 below it with $gray-500 color already establishes a color var at the end of that line and size. Is line 86 in parenthisis conditional to have CSS VAR || Color Var? If so then I guess you wouldn't need this line 85. If not, then this line 85 would be same as line 86 but with only the color var, right? and line 86 would just have the CSS var, correct?

Reply:
Good catch.  I'll verify which property is correct and update.